sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/yaml/TemplateUtilsTest.java

```diff
@@ -100,4 +100,55 @@ public void testApplyMissingExceptionDisableBlockPasses() throws IOException {
                 Collections.emptyMap(),
                 TemplateUtils.MissingBehavior.EXCEPTION));
     }
+
+    @Test
+    public void testApplyTrueEnvVarToSection() throws IOException {
+        String filename = "test-render-inverted.yml";
+        String yaml = getYamlContent(filename);
+        Assert.assertTrue(yaml.contains("ENABLED"));
+
+        Map<String, String> envMap = new HashMap<>();
+        envMap.put("ENABLED", String.valueOf(true));
+        String renderedYaml = TemplateUtils.applyEnvToMustache(
+                filename, yaml, envMap, TemplateUtils.MissingBehavior.EMPTY_STRING);
+
+        Assert.assertTrue(renderedYaml.contains("cmd: ./enabled true"));
+        Assert.assertFalse(renderedYaml.contains("ENABLED"));
```

Comment:
Should we check to ensure that `cmd: ./disabled true` is missing from this too? (also for `cmd: ./enabled false` in the test below)?